lib/rubocop/cop/offense.rb

```diff
@@ -146,7 +146,7 @@ def highlighted_area
       # @api private
       # This is just for debugging purpose.
       def to_s
-        format('%<severity>s:%3<line>d:%3<column>d: %<message>s',
```

Comment:
The following change leads to a [build failure on JRuby](https://app.circleci.com/pipelines/github/rubocop/rubocop/6553/workflows/3651e6dc-fe0b-4025-b079-384aa16557de/jobs/225855):

```diff
-        format('%<severity>s:%3<line>d:%3<column>d: %<message>s',
+        format('%{severity}:%3{line}:%3{column}: %{message}',
```

```
       expected: "C:  1:  1: message % test"
            got: "C:1:1: message % test"
```

Works fine on MRI. Even though it probably makes sense to file a JRuby ticket, I'd say that we have to keep this as is to support correct behaviour on older JRuby versions.

cc @headius